src/Vegetation/PlantHydraulics.jl

```diff
@@ -255,65 +264,65 @@ end
 Computes the effective saturation (volumetric water content relative to
 porosity).
 """
-function effective_saturation(ν::FT, ϑ_l::FT) where {FT}
-    S_l = ϑ_l / ν # S_l can be > 1
+function effective_saturation(plant_ν::FT, ϑ_l::FT) where {FT}
+    S_l = ϑ_l / plant_ν # S_l can be > 1
     safe_S_l = max(S_l, 0)
-    return safe_S_l # units of [m3 m-3]
+    return safe_S_l # (m3 m-3)
 end
 
 """
     water_retention_curve(
-        vg_α::FT,
-        vg_n::FT,
-        vg_m::FT,
+        plant_vg_α::FT,
+        plant_vg_n::FT,
+        plant_vg_m::FT,
         S_l::FT,
-        ν::FT,
-        S_s::FT) where {FT}
+        plant_ν::FT,
+        plant_S_s::FT) where {FT}
 Converts augmented liquid fraction (ϑ_l) to effective saturation 
 (S_l), and then effective saturation to absolute pressure (p), 
 for both the unsaturated and saturated regimes. Note pressure 
 is in SI units [m].
 """
 function water_retention_curve(
-    vg_α::FT,
-    vg_n::FT,
-    vg_m::FT,
+    plant_vg_α::FT,
+    plant_vg_n::FT,
+    plant_vg_m::FT,
     S_l::FT,
-    ν::FT,
-    S_s::FT,
+    plant_ν::FT,
+    plant_S_s::FT,
 ) where {FT}
     if S_l <= FT(1.0)
-        p = -((S_l^(-FT(1) / vg_m) - FT(1)) * vg_α^(-vg_n))^(FT(1) / vg_n)
+        ψ = -((S_l^(-FT(1) / plant_vg_m) - FT(1)) * plant_vg_α^(-plant_vg_n))^(FT(1) / plant_vg_n)
     else
-        ϑ_l = augmented_liquid_fraction(ν, S_l)
-        p = (ϑ_l - ν) / S_s
+        ϑ_l = augmented_liquid_fraction(plant_ν, S_l)
+        ψ = (ϑ_l - plant_ν) / plant_S_s
     end
-    return p # units of (m)
+    return ψ # (m)
 end
 
 """
     inverse_water_retention_curve(
-        vg_α::FT,
-        vg_n::FT,
-        vg_m::FT,
-        p::FT,
-        ν::FT,
-        S_s::FT) where {FT}
-Converts absolute pressure (p) to effective saturation (S_l).
+        plant_vg_α::FT,
+        plant_vg_n::FT,
+        plant_vg_m::FT,
+        ψ::FT,
+        plant_ν::FT,
+        plant_S_s::FT) where {FT}
+Converts absolute pressure (ψ) to effective saturation (S_l).
```

Comment:
not pressure anymore

Reply:
will change, thanks

Reply:
replaced by this comment, following notes in the clima docs :

Converts absolute pressure, represented by the height (ψ)
of the water column that would give rise to this pressure,
to effective saturation (S_l).

What do you think?

Reply:
sure!